core-bundle/src/Resources/contao/library/Contao/InsertTags.php

```diff
@@ -444,13 +444,13 @@ protected function doReplace($strBuffer, $blnCache)
 
 								if ($objNext instanceof PageModel)
 								{
-									$strUrl = \in_array('absolute', $flags, true) ? $objNext->getAbsoluteUrl() : $objNext->getFrontendUrl();
+									$strUrl = \in_array('absolute', \array_slice($elements, 2), true) || \in_array('absolute', $flags, true) ? $objNext->getAbsoluteUrl() : $objNext->getFrontendUrl();
```

Comment:
Do we really need the `array_slice($elements, 2)` everywhere? We know that `$elements[0]` is the name of the insert tag and `$elements[1]` is a numeric ID, so there should not be any false positives if we just used `in_array('absolute', $elements)`, should there?

Reply:
IMO it’s forwards compatibility with #3230

Reply:
But isn't the `array_slice()` redundant in #3230 as well? IMHO `$elements[0]` and `$elements[1]` will never be `absolute` or `blank`, so we do not have to use `array_slice()` at all.

Reply:
> IMHO `$elements[0]` and `$elements[1]` will never be `absolute` or `blank`,

They could if you have a page with the alias `absolute` or `blank`